mimic/rest/identity_admin_api.py

```diff
@@ -0,0 +1,102 @@
+# -*- test-case-name: mimic.test.test_identity_admin -*-
+"""
+Mocks for the identity admin API.
+"""
+from json import dumps, loads
+
+from zope.interface import implementer
+from twisted.plugin import IPlugin
+
+from mimic.imimic import IAPIMock
+from mimic.rest.mimicapp import MimicApp
+from mimic.model.identity_admin_objects import bad_request
+
+
+@implementer(IAPIMock, IPlugin)
+class IdentityAdminAPI(object):
+    """
+    A mock of the OpenStack Identity Admin API.
+    """
+    def catalog_entries(self, tenant_id):
+        """
+        Return the catalog entries for this tenant.
+        """
+        return []
+
+    def resource_for_region(self, region, uri_prefix, session_store):
+        """
+        Creates an identity admin resource.
+        """
+        return _IdentityAdminImpl().app.resource()
+
+
+class _IdentityAdminImpl(object):
+    """
+    Klein resources for the Identiy admin API.
+
+    TODO: come up with a way better name than IdentityAdminImpl
+    """
+    app = MimicApp()
+
+    @app.route("/v2.0/OS-KSCATALOG/endpointTemplates", methods=("POST",))
+    def add_endpoint_template(self, request):
+        """
+        Adds an endpoint template.
+        """
+        try:
+            content = loads(request.content.read())
+            content = content["OS_KSCATALOG:endpointTemplate"]
+        except (ValueError, KeyError):
+            request.setResponseCode(400)
+            return dumps(bad_request("Invalid JSON request body"))
+
+
+create_endpoint_template_schema = {
```

Comment:
This is unused. Perhaps useful in a future branch but we can remove it for now, since it does nothing here.